cmd/main.go

```diff
@@ -205,6 +205,11 @@ func main() {
 		setupLog.Error(err, "Failed to create gcp clients with sa json key path: "+env.Get("GCP_SA_JSON_KEY_PATH"))
 		os.Exit(1)
 	}
+	gcpVpcPeeringClients, err := gcpclient.NewGcpClients(ctx, env.Get("GCP_VPC_PEERING_KEY_PATH"), rootLogger.WithName("gcp-clients-vpcpeering"))
+	if err != nil {
+		setupLog.Error(err, "Failed to create gcp clients with sa json key path: "+env.Get("GCP_VPC_PEERING_KEY_PATH"))
+		os.Exit(1)
+	}
```

Comment:
try using the existing `gcpClients` already defined, and their implementation implement a specific token provider for your specific service account